public/src/books.js

```diff
@@ -1,5 +1,5 @@
 function findAuthorById(authors, id) {
-  return authors.find((author) => author.id === id);
+  return authors.find((author) => author.id == id);
```

Comment:
I had to change this in order to be able to use it in `home.js` against books! Apparently, the authorId on a book is a string, but is an integer on an author object itself.